src/tools/goid-tool-cmdline.c

```diff
@@ -1038,7 +1038,7 @@ void free_list(struct generic_list *list, short string_arg)
  * @brief updates a multiple option starting from the passed list
  */
 static
-void update_multiple_arg(void *field, char ***orig_field,
+int update_multiple_arg(void *field, char ***orig_field,
```

Comment:
I suggest to ignore the cmdline files and instead report this problem to gengetopt